cmake/Reprojucer.cmake

```diff
@@ -1625,6 +1635,42 @@ function(__set_bundle_properties target_name extension)
 endfunction()
 
 
+function(__set_plugin_output_directory_property target_name plugin_type plugins_dir)
+
+  if(CMAKE_GENERATOR STREQUAL "Xcode")
+    foreach(configuration_name ${JUCER_PROJECT_CONFIGURATIONS})
+      if(DEFINED JUCER_${plugin_type}_BINARY_LOCATION_${configuration_name})
+        set(install_path ${JUCER_${plugin_type}_BINARY_LOCATION_${configuration_name}})
+      else()
+        set(install_path "$ENV{HOME}/Library/Audio/Plug-Ins/${plugins_dir}")
+      endif()
+      string(APPEND all_confs_install_path
+        "$<$<CONFIG:${configuration_name}>:${install_path}>"
+      )
+    endforeach()
+
+    # See XCodeProjectExporter::Target::getTargetSettings()
+    # in JUCE/extras/Projucer/Source/Project Saving/jucer_ProjectExport_XCode.h
+    set_target_properties(${target_name} PROPERTIES
+      XCODE_ATTRIBUTE_INSTALL_PATH "${all_confs_install_path}"
+      XCODE_ATTRIBUTE_DEPLOYMENT_LOCATION "YES"
```

Comment:
Since I don't want to force users to change their CMake version when there are using `VST_BINARY_LOCATION` or `AU_BINARY_LOCATION` with the Xcode generator, I went back to the first approach, but making it much simpler.